bin/cloudgov/entrypoint.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/sh
+
+/var/www/vendor/bin/drush state:set system.maintenance_mode 0
+/var/www/vendor/bin/drush pm:uninstall usagov_login || true
+/var/www/vendor/bin/drush pm:enable usagov_benefit_finder_content
+/var/www/vendor/bin/drush pm:enable usagov_benefit_finder_api
+/var/www/vendor/bin/drush pm:enable usagov_benefit_finder
+/var/www/vendor/bin/drush pm:enable usagov_benefit_finder_page
+/var/www/vendor/bin/drush cr
+cd /var/www/web/
+mkdir -p content/sync
```

Comment:
is this related to the content sync module work we attempted, or something else? still needed?

Reply:
/var/www/vendor/bin/drush pm:enable usagov_benefit_finder_content
/var/www/vendor/bin/drush pm:enable usagov_benefit_finder_api
/var/www/vendor/bin/drush pm:enable usagov_benefit_finder
/var/www/vendor/bin/drush pm:enable usagov_benefit_finder_page
Do not need these now because the modules are enabled now in database.

Reply:
@gchi25 @scottqueen-bixal Removing all these.